schemas/test/en/test_supplementary_data_non_list_items.json

```diff
@@ -0,0 +1,503 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "123",
+    "title": "Test Supplementary Data with non list items",
+    "theme": "default",
+    "description": "A questionnaire to demo loading Supplementary data and using non list items for value sources.",
+    "metadata": [
+        {
+            "name": "survey_id",
+            "type": "string"
+        },
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        },
+        {
+            "name": "sds_dataset_id",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Hub",
+        "options": {
+            "required_completed_sections": ["introduction-section"]
+        }
+    },
+    "sections": [
```

Comment:
Should add a repeating section and a repeating block to demo you can pipe in non list item data inside repeating blocks/sections.

I don't think the schema needs to be `non_list_items`. For list items, we will need repeats, so we can extend the same survey instead of duplicating most of it when we support list item values. Thoughts?

Reply:
Good idea - will rename so it can just be extended and add in the repeating section

Reply:
Right now it doesn't seem to check whether list item id should be used which means inside a repeating section/block, non-repeating data won't work. You would need the similar check as answers sources to work out if we are in repeating section etc before using the list item id.

Reply:
Renamed to `all_variations` (but open to a better name if anyone has one), and its now got a repeating section.

The data is working in a repeating section because it doesn't make use of list item id, so at the moment its the other way round, in a repeating section repeating data will not work. I can look to add these checks and test but thinking that'd probably be more within the scope of the data with list items ticket?

Reply:
Do we even need `all_variantions`?.

> The data is working in a repeating section because it doesn't make use of list item id, so at the moment its the other way round, in a repeating section repeating data will not work. I can look to add these checks and test but thinking that'd probably be more within the scope of the data with list items ticket?

Sounds good. Happy for it to be tackled in the next card. 👍

Reply:
I'd ditch it but we have an empty schema with that name, and the mocking for tests like `test_session` gets a bit ugly if I use the new schema as it can't handle the introduction block, maybe that one could be `_empty` or actually - if I just enable the hub from the start instead of after the intro that might also sort it

Reply:
Im a bit confused as to why we need the empty one?.

Reply:
It made the tests that don't care about the actual supplementary data simpler - but I agree, I'll get rid of it and fix the tests